src/components/marked/mini-graphiQL.tsx

```diff
@@ -256,7 +256,11 @@ class QueryEditor extends Component {
   }
 
   render() {
-    return <div className="query-editor" ref={e => (this.domNode = e)} />
+    return (
+      <div className="query-editor" ref={e => (this.domNode = e)}>
+        <span className="editor-name">Operation</span>
```

Comment:
Technically this is a "document" since it may contain fragments and multiple operations; but I think that's probably a little too technical for people learning the technology so I think you've made the right choice of term here :+1: